### PR TITLE
feat: Adding guide to 'Run Your Own Code in a Reactor'

### PR DESCRIPTION
**Pull request by dhudec**
## Description

- Adds guide: 'Run Your Own Code in a Reactor'

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

(full screen screenshot messed up the header)
image

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: _sass/custom/nav.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   .side-bar-category-items.open {
     opacity: 1;
-    max-height: 550px;
+    max-height: 750px;
```
Comment (dhudec) [inline _sass/custom/nav.scss:15]: We have enough content that we reached this max-height! This causes the Reference section to overlap the Guides list in the sidebar (🤮 ). I just increased this max-height a bit to fix this, but once we really have too much content to show, we'll need to spend a bit more time figuring out how to scroll the overflow.

File: collections/guides/use/run-your-own-code-in-a-reactor.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+      },
+      metadata: {
+        lithic_token_id: lithicCard.token
+      }
+    }
+  };
+}
+```
+
+Next, we need to create a Private Reactor Formula in Basis Theory containing this `code`, `configuration`, and `request_parameters` by calling the
```
Comment (nloding) [inline collections/guides/use/run-your-own-code-in-a-reactor.md:249]: i personally found this step confusing when creating my first custom reactor. having to stringify the function for the `code` property, listing all the args and config values, etc. i think an example of the Lithic formula object might be a good idea to show here?

File: collections/guides/use/run-your-own-code-in-a-reactor.md [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+
+Many third party APIs or SDKs return errors in unique ways, so Reactor Formulas are expected to catch and handle any errors that occur during execution. 
+Any errors that occur during the execution of your Reactor Formula should be caught, translated into a standard Basis Theory [Reactor error](https://github.com/Basis-Theory/basistheory-reactor-formulas-sdk-js/tree/master/src/errors),
+and this standardized error rethrown from the formula.
+
+The standard Basis Theory [Reactor errors](https://github.com/Basis-Theory/basistheory-reactor-formulas-sdk-js/tree/master/src/errors) 
+are defined within the NPM package [@basis-theory/basistheory-reactor-formulas-sdk-js](https://www.npmjs.com/package/@basis-theory/basis-theory-reactor-formulas-sdk-js),
+which is available to import from within your Reactor Formulas.
+These standard error types will be translated by the Reactor execution environment into [HTTP errors](https://docs.basistheory.com/#errors-reactor-errors)
+returned from the Basis Theory API. Any other uncaught exceptions thrown from a Reactor Formula will be returned as an HTTP `500` error.
```
Comment (brigonzalez) [inline collections/guides/use/run-your-own-code-in-a-reactor.md:142]: maybe we mention something about retries on certain errors?

Comment (koalaty-code) [inline collections/guides/use/run-your-own-code-in-a-reactor.md:142]: Do we want to call out that the original error message will be lost and not returned in the generic 500 response in this scenario?

File: collections/guides/use/run-your-own-code-in-a-reactor.md [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+that instruct our Reactor to issue a card called `John's Travel Card` with a total spending limit of $500.
+
+```bash
+curl 'https://api.basistheory.com/reactors/ab8a1c9a-8666-449d-91c7-6c0d77bf7d02/react' \
+  -X 'POST' \
+  -H 'BT-API-KEY: key_PtkNaBwXe23rGt1eZenMMx' \
+  -H 'Content-Type: application/json' \
+  -d '{
+      "args": {
+        "memo": "John'\''s Travel Card",
```
Comment (brigonzalez) [inline collections/guides/use/run-your-own-code-in-a-reactor.md:310]: does `John's Travel Card` parse correctly? just interesting that we have to escape the single quote with the single quotes around it

Comment (brigonzalez) [reply collections/guides/use/run-your-own-code-in-a-reactor.md:310]: nvm. looks like it does. I missed this whole thing being wrapped in single quotes. confusing

Comment (dhudec) [reply collections/guides/use/run-your-own-code-in-a-reactor.md:310]: It looks strange, but it works - I copied this out of postman's curl translation, and I tried removing the extra quotes but it becomes an invalid command without them

File: collections/guides/use/run-your-own-code-in-a-reactor.md (hunk 2 above)
Comment (dhudec) [reply collections/guides/use/run-your-own-code-in-a-reactor.md:249]: Oh, man... I would really like to add this example, but the curl request is 86 lines long... is it worth taking up that much space?

Comment (dhudec) [reply collections/guides/use/run-your-own-code-in-a-reactor.md:250]: I added this example - I got it to be a bit shorter but it's still quite long and with a lot of side scrolling to side the code string

**Review by koalaty-code — APPROVED** [review]

File: collections/guides/use/run-your-own-code-in-a-reactor.md (hunk 2 above)
Comment (nloding) [reply collections/guides/use/run-your-own-code-in-a-reactor.md:250]: @dhudec i think you could do something like `"code": "module.exports = async function (req) {\n ...` as you have the "Note" above.

**Pull request merged**